utils/submission.js

```diff
@@ -2,95 +2,99 @@ const { getResponseJSON, setHeaders, logIPAddress } = require('./shared');
 const fieldMapping = require('./fieldToConceptIdMapping');
 
 const submit = async (res, data, uid) => {
-    
     // Remove locked attributes.
     const { lockedAttributes } = require('./shared');
     lockedAttributes.forEach(atr => delete data[atr]);
 
-    // generate Connect_ID if Consent form submitted
-    if(data[919254129] !== undefined && data[919254129] === 353358909) {
-        const { generateConnectID } = require('./shared');
-        const { sanityCheckConnectID } = require('./firestore');
-        let boo = false;
-        let Connect_ID;
-        while(boo === false){
-            const ID = generateConnectID();
-            const response = await sanityCheckConnectID(ID);
-            if(response === true) {
-                Connect_ID = ID;
-                boo = true;
+    try {
+        // generate Connect_ID if Consent form submitted
+        if(data[919254129] !== undefined && data[919254129] === 353358909) {
+            const { generateConnectID } = require('./shared');
+            const { sanityCheckConnectID } = require('./firestore');
```

Comment:
I feel that multiple rounds of imports from `shared.js` and `firestore.js` slows down code. It could be faster if imports are done at top of file.